esp/esdllib/README.md

```diff
@@ -238,50 +258,39 @@ These both append the value of the `@select` node to the value of the `@target`
-------|--------------------------------------------------|-------------------
-  1   | Reciept of request from client                   |
-  2   | Authentication and Authorization                 |
-  3   | ESDL Transforms ESP Request to backend request   |
-  4   | | **BackendRequest** : Transform backend request to final form
-  5   | Backend is called, sending backend request       | **Service** : A script is run instead of calling out to the backend
-  6   | Backend response received                        |
-  7   | | **BackendResponse** : Transform backend response
-  8   | ESDL Transforms backend response to initial ESP response |
-  9   | | **ESDL Response** : Transform initial ESP response to final ESP response
-  10  | ESDL Transforms final ESP response to match schema |
-  11  | ESP sends final response to client |
-  12  | | **PreLogging** : Transform Log Request
-  13  | Logging Manager dispatches Log Request
-
-A script is inserted to an entry point by naming its root node after the name of the entry point. A script can replace a backend service call by configuring it as a **Service** entry point.
-
-Additional details about the entry points and script configuration is found in the reference section.
-
-### Illustrated by Flowchart
-
-The lifecycle of an ESP transaction is shown below in the flowchart's blue boxes. Optionally, scripts can be inserted at any of the entry points named in green boxes. At each entry point the script has the opportunity to transform the source in preparation for passing it on to the next transaction step.
-
-![ESP Transaction Flowchart with ESDL Script Entry Points](readme_assets/ESDLScriptEntryPoints.png)
+The lifecycle of an ESP transaction is shown below in the flowchart's plain black boxes. Optionally, scripts can be inserted at any of the entry points named in the light grey boxes. At each entry point the script has the opportunity to transform the source in preparation for passing it on to the next transaction step.
+
+``` mermaid
+flowchart TD
+    classDef transactionStep fill:#000;
+    classDef entryPoint fill:#444;
+    start(( ))-->receive[Receive request]
+    receive-->esdl1[ESDL transforms ESP request \n to backend request]
+    esdl1-->backendRequest[[BackendRequest:\nTransform backend request\nto final form]]
+    backendRequest-->isScript{Script\nbackend?}
+    isScript--no-->callBackend[Call backend]
+    isScript--yes-->service[[Service: \nRun script as backend,\n create output]]
+    callBackend-->backendResponse[[BackendResponse: \nTransform backend response]]
+    service-->backendResponse
+    backendResponse-->esdl2[ESDL transforms backend response to \n initial ESP response]
+    esdl2-->esdlResponse[[EsdlResponse: \n Transform initial ESP response \n to final ESP response]]
+    esdlResponse-->esdl3[ESDL transforms final ESP \nresponse to match schema]
+    esdl3-->preLogging[[PreLogging: \n Transform Log Request]]
```

Comment:
I'd like to avoid going into that level of detail here, but it looks like the response is sent back to the caller then the esdlService->handleResultLogging() method is called. If someone has requirements like that, I think it would be best for us to conclusively research and document at that time.